stress/deployment.template.json

```diff
@@ -29,6 +29,11 @@
             "type": "docker",
             "status": "running",
             "restartPolicy": "always",
+            "env": {
```

Comment:
Should we also configure Edge Hub to log to Influx? And provide the Influx co-ordinates?

Reply:
Talked offline that the default configuration will work since it will connect to a server named `influxdb` which is a module that gets deployed here.